src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/controller/TierCalculationRequiredEventListener.kt

```diff
@@ -14,6 +15,12 @@ class TierCalculationRequiredEventListener(
   private val calculator: TierCalculationService
 ) {
 
+  @MessageExceptionHandler()
```

Comment:
This will hopefully help us investigate why calculations have failed

Reply:
noice